message-processor/src/main/java/com/bosch/iot/things/example/message/processor/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.bosch.iot.things.example.message.processor;
+
+public class Constants {
+    public static final String TELEMETRY_ENDPOINT = "telemetry/";
+    public static final String EVENT_ENDPOINT = "event/";
+    public static final String CONTROL_ENDPOINT = "control/";
```

Comment:
`control` endpoint is deprecated, replace with `command`/`command_response`

Reply:
Yes, I did notice some changes must have been made recently, as some things didn't work accordingly!
Will have a look at the new endpoints and make the necessary adjustments!
thanks